src/_locales/en_US/messages.json

```diff
@@ -199,9 +199,17 @@
         "message": "You're now protected by Privacy Badger. To learn how Privacy Badger works, click below for a quick tutorial.",
         "description": ""
     },
+    "options_incognito_setting": {
+        "message": "Learn in Private/Incognito windows **",
+        "description": "Checkbox label on the general settings page"
+    },
+    "options_incognito_warning": {
+        "message": "** Learning which domains are trackers in Private/Incognito windows will leave traces of your private browsing history on your computer. By default, Privacy Badger will still block trackers it already knows about in incognito windows - but you can enable this option to allow it to learn about new trackers and store information on them too.",
```

Comment:
How about the following?
>Enabling learning in Private/Incognito windows may leave traces of your private browsing history on your computer. By default, Privacy Badger will block trackers it already knows about in Private/Incognito windows, but it won't learn about new trackers. You might want to enable this option if most of your browsing happens in Private/Incognito windows.

Reply:
I used this but replaced "most of your browsing" with "a lot of browsing" at the end. What do you think?
I don't want this option to sound scary or make people wonder if enough of their browsing happens in Incognito mode. It's really a question of whether you're using a shared computer or not.